.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,13 +3,13 @@ Resolves
 
 ## Submitter Checklist:
 
+- [ ] Requested a security/privacy review as needed
```

Comment:
minor suggestion: `as needed` should link to https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed

Reply:
@diracdeltas `brave/handbook` is not public, do we want to link to that in the PR template?

Reply:
We could probably move/copy the first part of that section to the public `brave/brave-browser` wiki.